music21/note.py

```diff
@@ -1611,9 +1627,6 @@ class Unpitched(NotRest):
     A General class of unpitched objects which appear at different places
     on the staff.  Examples: percussion notation.
 
-    The `Unpitched` object does not currently do anything and should
-    not be used.
-
```

Comment:
can we change docs below to "which shows where they should be displayed as if the staff were a 5-line staff in treble clef"?